.github/config.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Heavily inspired by https://github.com/electron/electron/blob/master/.github/config.yml
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
```

Comment:
Maybe this should be similar to the pr text `👋 Thanks for opening your first issue here! 👋` and then the rest on the next lines